bio/minibwa/map/wrapper.py

```
@@ -0,0 +1,154 @@
+
+expected_sort_orders = {"coordinate", "queryname", None}
+sort_order = snakemake.params.get("sort_order", "coordinate")
+if sort_order not in expected_sort_orders:
+    raise ValueError(
+        "Unexpected value for `params.sort_order`. "
+        f"Expected one of {expected_sort_orders}, got `{sort_order}`."
+    )
+
+# Extract index from input
+# and check that all required indices are declared
+index_prefix = commonprefix([str(o) for o in snakemake.input.index]).strip(".")
```

Ruff (0.15.21)

[error] Undefined name `snakemake`

(F821)